packages/@aws-cdk/aws-applicationautoscaling/lib/schedule.ts

```diff
@@ -17,6 +17,13 @@ export abstract class Schedule {
    * Construct a schedule from an interval and a time unit
    */
   public static rate(duration: Duration): Schedule {
+    const validDurationUnit = ['minute', 'minutes', 'hour', 'hours', 'day', 'days'];
+    if (validDurationUnit.indexOf(duration.unitLabel()) === -1) {
+      throw new Error('Allowed unit for scheduling is: \'minute\',\'minutes\', \'hour\', \'hours\', \'day\' or \'days\'');
```

Comment:
Some code readability.

```suggestion
    if (!validDurationUnit.includes(duration.unitLabel())) {
      throw new Error("Allowed unit for scheduling is: 'minute', 'minutes', 'hour', 'hours', 'day' or 'days'");
```